lib/src/rules/use_late_for_private_fields_and_variables.dart

```diff
@@ -177,9 +181,9 @@ class _Visitor extends UnifyingAstVisitor<void> {
 
   void _checkAccess(Iterable<CompilationUnitElement> units) {
     final allNullableAccess =
-        units.expand((unit) => nullableAccess[unit]!).toSet();
+        units.expand((unit) => nullableAccess[unit] ?? {}).toSet();
```

Comment:
Consider making empty collections that are never stored (and hence never added to) be `const`.

Reply:
Good catch!